multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -23,6 +23,9 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
```

Comment:
@pbx0 the lexical ordering on the names of the dropin files determines it. First the unit definition is processed, followed by drop-in files in lexical order by file name. 

With `ExecStart` or `Environment` overrides, it's the last (most recently encountered) definition that wins when the unit is evaluated.